classrooms/models.py

```
@@ -1,3 +1,82 @@
+
+    published = models.BooleanField(default=False)
+    upcoming = models.BooleanField(default=False)
+    published_on = models.DateTimeField(null=True, blank=True)
+    paid_course = models.BooleanField(default=False)
+    course_price = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
+   
+    slug = models.SlugField(max_length=255, unique=True)
+    total_rating = models.DecimalField(max_digits=5, decimal_places=2, default=0.0)
+    number_of_ratings = models.PositiveIntegerField(default=0)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)